# Statistic ICSC303 Final Exam
### Name: Vikrom Narula ID: 6081050

## Question 1
The National Center for Health Statistics interviewed 5409 adult smokers in 2020, and 2636
of them said they had tried to quit smoking during the past year. Consider this to be a
random sample.

a) Find a 95% confidence interval for the proportion of smokers who have tried
to quit within the past year.

In [7]:
quit <- 2636 / 5409

alpha <- 0.05
len <- 5409
df <- len - 1

z <- qt(1 - alpha / 2, df=df)

standE <- sqrt((quit * (1 - quit)) / len)

cat("Confidence Interval:", quit + c(-1, 1) * z * standE)

Confidence Interval: 0.4740124 0.5006594

b) Find a 99% confidence interval for the proportion of smokers who have tried
to quit within the past year.

In [8]:
quit <- 2636 / 5409

alpha <- 0.01
len <- 5409
df <- len - 1

z <- qt(1 - alpha / 2, df=df)

standE <- sqrt((quit * (1 - quit)) / len)

cat("Confidence Interval:", quit + c(-1, 1) * z * standE)

Confidence Interval: 0.4698236 0.5048482

c) Someone claims that the proportion is less than 0.5. With what level of
confidence can this statement be made?

Answer: 99%

## Question 2
An experiment was conducted to test the effect of a new drug on a viral infection. The
infection was induced in 100 mice, and the mice were randomly split into two groups of 50.
The first group, the control group, received no treatment for the infection. The second group
received the drug. The proportions of survivors, and , in the two groups after a 30-day
period, were found to be 0.40 and 0.64, respectively. Is there sufficient evidence to indicate
that the drug is effective in treating the viral infection? Use α = 0.05.

a) Compute the test statistic.

In [18]:
len <- 50
control <- 0.4
controlK <- 0.4 * len
treatment <- 0.64
treatmentK <- 0.64 * len 



z <- (control - treatment) / sqrt(((controlK + treatmentK) / 100)*(1 - (controlK + treatmentK) / 100)*(2 / len))
z

[1] -2.401922

b) What is the critical value?

In [19]:
cat("Critical Value(s) =", qt(0.05, df=len - 1))

Critical Value(s) = -1.676551

c) What is the conclusion?

$| H_a |$ > $| H_0 |$ Hence we reject null hypothesis cause we have to check for $| H_a |$ $\ge$ $| H_0 |$

## Question 3
A national survey states that 67% of college students are under the age of 25, 21% are
between the ages of 25 and 30, 8% are between 30 and 40, and 4% are over 40. A random
sample of 250 students at Grand Rapids Community College yielded the following data:

In [23]:
data = data.frame(
  "Statistics"= c("Under 25", "25 - 29", "30 - 40", "40+"),
  "Freq"= c(138 , 62, 32, 18)) 

data

Statistics Freq
1 Under 25   138 
2 25 - 29     62 
3 30 - 40     32 
4 40+         18

a) Set up null and alternative hypotheses.

$H_0$: The college student ages agree with national survey. $p_1 = 0.67, p_2 = 0.21, p_3 = 0.08, p_4 = 0.04$;
$H_a$: The college student ages disagree with national survey.

b) Compute the value of the test statistic $X^2$

In [24]:
mtx = matrix(c(138 , 62, 32, 18), nrow=4, byrow=T)
mtx

chisq.test(mtx)

[,1]
[1,] 138 
[2,]  62 
[3,]  32 
[4,]  18


	Chi-squared test for given probabilities

data:  mtx
X-squared = 137.78, df = 3, p-value < 2.2e-16


c) What is the p-value associated with the test?

p-value = 2.2e-16

d) What is the appropriate conclusion for α = 0.1?

We reject the null hypothesis cause the p-cal is lesser than 0.1

## Question 4
A study was conducted to determine whether opinion concerning the addition of seat belts in
school buses is independent of the population density in which a person resides. The
following data was recorded:

In [26]:
data = data.frame(
  "Statistics"= c("Favor", "Oppose", "Undecied"),
  "Rural"= c(75, 40, 13),
  "Suburban"= c(62, 34, 18),
  "Urban"= c(87, 39, 12)
) 
data

Statistics Rural Suburban Urban
1 Favor      75    62       87   
2 Oppose     40    34       39   
3 Undecied   13    18       12

a) Set up null and alternative hypotheses.

$H_0$: opinion concerning the addition of seat belts in school buses is dependent of the population density in which a person resides.;
$H_a$: opinion concerning the addition of seat belts in school buses is independent of the population density in which a person resides.

b) Compute the value of the test statistic $X^2$

In [28]:
mtx = matrix(c(75, 40, 13, 62, 34, 18, 87, 39, 12), nrow=3)
mtx

chisq.test(mtx)

[,1] [,2] [,3]
[1,] 75   62   87  
[2,] 40   34   39  
[3,] 13   18   12


	Pearson's Chi-squared test

data:  mtx
X-squared = 4.0034, df = 4, p-value = 0.4055


c) What is the p-value associated with the test?

p-value = 0.4055

d) What is the conclusion of the test?

If given that alpha is 0.5 the p-value is lower so we rejec the null hypothesis and agree that opinion concerning the addition of seat belts in school buses is independent of the population density in which a person resides.

## Question 5
The following measurements were recorded for the number of hours spent studying for a
statistics exam selected from a class:

32, 42, 40, 28, 22, 16, 18, 24, 36, 14, 30, 35, 42, 26, 30

5.1) Find the sample mean and the standard deviation of the sample mean.

In [35]:
data <- c(32, 42, 40, 28, 22, 16, 18, 24, 36, 14, 30, 35, 42, 26, 30)
mu <- mean(data)
SD <- sd(data)

cat("Mean:",mu,", Standard Diviation:", SD)

Mean: 29 , Standard Diviation: 9.079018

5.2) Find the 99% confidence interval for the average number of hours spent
studying for the exam for students in this class assuming that the number of hours are
normally distributed.

In [42]:
t.test(data, mu=mean, conf.level=0.99)

## Critical Value qt(alpha, df)
cat("Critical Value(s) =", qt(0.05, df=50))


	One Sample t-test

data:  data
t = 0, df = 14, p-value = 1
alternative hypothesis: true mean is not equal to 29
99 percent confidence interval:
 22.02171 35.97829
sample estimates:
mean of x 
       29 


Critical Value(s) = -1.675905

5.3) Test the hypothesis if the average time spent studying for the exam is more
than 30 hours at the significance level 0.05.

In [44]:
t.test(data, mu=mean, conf.level=0.95)

## Critical Value qt(alpha, df)
cat("Critical Value(s) =", qt(0.05, df=50))


	One Sample t-test

data:  data
t = 0, df = 14, p-value = 1
alternative hypothesis: true mean is not equal to 29
95 percent confidence interval:
 23.97221 34.02779
sample estimates:
mean of x 
       29 


Critical Value(s) = -1.675905

5.3.1) The null hypothesis is the average time spent studying for exam is less then 30 hours

5.3.2) The alternative hypothesis is the average time spent studying is 30 hours or more

5.3.3) The calculated test statistics is 0

5.3.4) The critical region is greater 1.675905

5.3.5) Decision Do not reject cause t is lesser

5.3.6) The average time spent studying for exam is less then 30 hours

## Question 6
### 6.1
The following data, recorded in hours, represent the length of time a
recovery for patients from the symptom after taking the one of the two medicines.

In [46]:
data = data.frame(
  "Statistics"= c("Sample Size", "Sample Mean", "Sample Standard Deviation"),
  "Medicine 1"= c(16, 6.6 ,1.5),
  "Medicine 2"= c(25, 7.2, 2.1)
) 
data

Statistics                Medicine.1 Medicine.2
1 Sample Size               16.0       25.0      
2 Sample Mean                6.6        7.2      
3 Sample Standard Deviation  1.5        2.1

Find the 99% confidence interval of the difference between recovery times of the two
medicines.

In [49]:
alpha <- 0.01

SE <- sqrt(1.5^2 / 16 + 2.1^2 / 25)
tHalf <- qt(1 - alpha / 2, df=16 - 1)
cat("Confidence Interval:", (6.6 - 7.2) + c(-1, 1) * tHalf * standE)

Confidence Interval: -7.507662 6.307662

### Question 6.2
In this situation, we ask the same patients to take both medicines on different
days. The following data present the number of hours each patient needs to recovery from
the symptom after taking the two medicines.

In [55]:
data = data.frame(
  "Statistics"= c("1", "2", "3", "4"),
  "Medicine 1"= c(3.6, 6.2, 4.8, 8.2),
  "Medicine 2"= c(4.2, 7.4, 4.6, 7.5)
) 
data

summary(data)

Statistics Medicine.1 Medicine.2
1 1          3.6        4.2       
2 2          6.2        7.4       
3 3          4.8        4.6       
4 4          8.2        7.5

 Statistics   Medicine.1    Medicine.2   
 1:1        Min.   :3.6   Min.   :4.200  
 2:1        1st Qu.:4.5   1st Qu.:4.500  
 3:1        Median :5.5   Median :6.000  
 4:1        Mean   :5.7   Mean   :5.925  
            3rd Qu.:6.7   3rd Qu.:7.425  
            Max.   :8.2   Max.   :7.500  

6.1.1) The null hypothesis is their is no diffrence in the medicine

6.1.2) The alternative hypothesis is their is a diffrence in the medicine

6.1.3) The calculated test statistics is

In [56]:
alpha <- 0.05

med1 <- c(3.6, 6.2, 4.8, 8.2)
med2 <- c(4.2, 7.4, 4.6, 7.5)

mean1 <- mean(med1)
mean2 <- mean(med2)

sd1 <- sd(med1)
sd2 <- sd(med2)

len1 <- length(med1)
len2 <- length(med2)

SE <- sqrt(sd1^2 / len1 + sd2^2 / len2)
tCal <- (mean1 - mean2) / SE

print(paste0("tCal =", tCal, ", tHalf =", qt(1 - (alpha / 2), min(len1 - 1, len2 - 1))))

[1] "tCal =-0.169650343047478, tHalf =3.18244630528371"


6.1.4) The critical region is 3.18244630528371

6.1.5) Decision Don not reject cause tcal < tHalf

## Question 7
The table below represents weights (kilograms) per plot of coffee beans planted using
different fertilizers. Test the hypothesis at the significant level 0.05 if there is no difference
among different types of fertilizers.

In [58]:
data = data.frame(
  "Statistics"= c("1", "2", "3", "4"),
  "T1"= c(220,160,240,180),
  "T2"= c(200,240,140,160),
  "T3"= c(160,170,200,140)
) 
data

Statistics T1  T2  T3 
1 1          220 200 160
2 2          160 240 170
3 3          240 140 200
4 4          180 160 140

7.1) The null hypothesis is theirs is diffrence between fertilizer

7.2) The alternative hypothesis is theirs is no diffrence between fertilizer

7.3) The F stat

In [59]:
## Combine all group
combineG <- data.frame(
    T1=c(220,160,240,180), 
    T2=c(200,240,140,160), 
    T3=c(160,170,200,140))

summary(combineG)

## Do the AVONA test
answer <- aov(values ~ ind,data = stack(combineG))
summary(answer)

       T1            T2            T3       
 Min.   :160   Min.   :140   Min.   :140.0  
 1st Qu.:175   1st Qu.:155   1st Qu.:155.0  
 Median :200   Median :180   Median :165.0  
 Mean   :200   Mean   :185   Mean   :167.5  
 3rd Qu.:225   3rd Qu.:210   3rd Qu.:177.5  
 Max.   :240   Max.   :240   Max.   :200.0  

            Df Sum Sq Mean Sq F value Pr(>F)
ind          2   2117    1058   0.809  0.475
Residuals    9  11775    1308               

7.4) p-value 0.475

7.5) Decision Do no reject cause p < 0.05 hecne do  not rteject
Conculde  theirs is no diffrence between fertilizer